adagrams/game.py

```diff
@@ -1,11 +1,95 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2: ["D", "G"], 
+    3: ["B", "C", "M", "P"], 
+    4: ["F", "H", "V", "W", "Y"], 
+    5: ["K"], 
+    8: ["J", "X"], 
+    10: ["Q", "Z"]
+    }
+
 def draw_letters():
-    pass
+    hand_of_letters = []
+    
+    while len(hand_of_letters) <= 9:
+        letter = random.choice(list(LETTER_POOL))
+        if hand_of_letters.count(letter) < LETTER_POOL[letter]:
+            hand_of_letters.append(letter)
+        if len(hand_of_letters) >= 10:
+            break
```

Comment:
This conditional says to `break` out of the loop if our `hand_of_letters` grows to 10 or bigger. However, your initial loop, `while len(hand_of_letters) <= 9` says the same thing! It turns out that if we delete these two lines of code, the tests still pass.